war/src/main/java/io/lumeer/engine/rest/BatchService.java

```diff
@@ -70,8 +70,8 @@
 
    @PostConstruct
    public void init() {
-      organizationFacade.setOrganizationId(organisationId);
-      projectFacade.setCurrentProjectId(projectId);
+      organizationFacade.setOrganizationCode(organisationId);
+      projectFacade.setCurrentProjectCode(projectId);
```

Comment:
`organizationId` and `projectId` attributes in this and other service classes need to be renamed to `organizationCode` and `projectCode` (or just `organization` and `project`).